main.js

```diff
@@ -49,8 +49,9 @@ function displayGrid() {
       </footer>
     </article>`;
     allIdeas += ideaCardHTML;
-    cardGrid.innerHTML = allIdeas;
   }
+  
+  cardGrid.innerHTML = allIdeas;
```

Comment:
This is such a little thing that caused such a headache. It makes me feel REALLY DUMB. Thanks for fixing it.